stdlib/hashmap.mc

```diff
@@ -30,12 +30,12 @@ let _hashmapBucketIdx = lam hash. lam hm. modi (absi hash) (length hm.buckets)
 
 
 -- 'hashmapEmpty' is an empty hashmap with a default number of buckets.
-let hashmapEmpty : HashMap =
+let hashmapEmpty : HashMap k v =
   {buckets = makeSeq _hashmapDefaultBucketCount [],
    nelems = 0}
 
 -- 'hashmapStrTraits' is traits for a hashmap with strings as keys.
-let hashmapStrTraits : HashMapTraits =
+let hashmapStrTraits : HashMapTraits String v =
```

Comment:
Should be just `HashMapTraits String` I guess

Reply:
Yes, fixed it now!